task_01/src/szyszki/task1.java

```diff
@@ -0,0 +1,46 @@
+package szyszki;
+import java.util.*;
+import java.lang.*; 
+
+public class task1 {
+
+	public static void main (String[] args)
+{
+		//
+		int a, b, c;
+		try 
+		{
+			a = Integer.parseInt(args[0]);
+			b = Integer.parseInt(args[1]);
+			c = Integer.parseInt(args[2]);
+		}
+		catch (Exception a)
+		{
+			System.out.println("Exception - something went wrong");

```

Comment:
Jak robimy try catch, i w bloku try coś nam nie wypali, to mamy dwie opcje. Albo to naprawimy, albo rzucamy wyjątek. Ty tutaj tylko piszesz coś na wyjściu i program stara się działać dalej. Jeżeli blok catch coś wyłapie, to rzucaj wyjątek. Stwórz nową klasę wyjątku i ją rzuć.